core/lib/Cardano/Address/Script/Parser.hs

```diff
@@ -72,6 +76,12 @@ scriptFromString str =
 --
 -- 5. Nested script are supported
 -- at_least 1 [3c07030e36bfffe67e2e2ec09e5293d384637cd2f004356ef320f3fe, all [3c07030e36bfffe67e2e2ec09e5293d384637cd2f004356ef320f3f1, 3c07030e36bfffe67e2e2ec09e5293d384637cd2f004356ef320f3f1]]
+-- 6. 1 signature required after slot number 120
+-- all [3c07030e36bfffe67e2e2ec09e5293d384637cd2f004356ef320f3fe, valid_from 120]
+-- 7. 1 signature required until slot number 150
+-- all [3c07030e36bfffe67e2e2ec09e5293d384637cd2f004356ef320f3fe, valid_until 150]
```

Comment:
How to lock your token permanently 1-o-1 :smile:

Reply:
well easy ;-) :